drivers/hud_apr/app/models/hud_apr/generators/shared/fy2020/question_six.rb

```diff
@@ -258,7 +258,8 @@ def self.question_number
       # veteran status
       answer = @report.answer(question: table_name, cell: 'B2')
       members = universe.members.where(
-        a_t[:veteran_status].in([nil, 8, 9]).
+        a_t[:veteran_status].in([8, 9]).
+          or(a_t[:veteran_status].eq(nil)).
           or(a_t[:veteran_status].eq(1).
```

Comment:
Why is this looking at 8, 9, 1 or nil?, isn't that the same as any veteran status?

Reply:
I'll adjust the formatting to make it clearer -- but this is either [nil, 8, 9], or 1 and age < 18

Reply:
Got it!